client/my-sites/plans-features-main/index.jsx

```diff
@@ -434,6 +439,41 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	getProductsSelectorSubHeader() {
+		const { purchases, siteSlug, translate } = this.props;
+		const baseCopy = translate( "Just looking for a backups? We've got you covered." );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Just looking for backups? We’ve got you covered.' )` *ES Score: 6*